pyproject.toml

```diff
@@ -30,20 +29,20 @@ keywords = [
 dynamic = ['version']
 dependencies = [
     "attrs>=22.2.0",
-    "botorch>=0.8.1",
+    "botorch>=0.9.3",
     "cattrs>=23.2.0",
     "exceptiongroup",
     "funcy>=1.17",
     "gpytorch>=1.9.1",
     "ngboost>=0.3.12",
     "numpy>=1.24.1",
-    "pandas[parquet]>=1.4.2",
+    "pandas>=1.4.2",
```

Comment:
@AdrianSosic I've made this additional change, if I'm not mistaken your change to the data frame serialization means we don't need `parquet` anymore, right?

Reply:
Correct, thanks for the catch!